src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngine.cs

```diff
@@ -386,12 +362,12 @@ public string GetAbsolutePath(string executingFilePath, string pagePath)
         {
             // Only use the area view location formats if we have an area token.
             var viewLocations = !string.IsNullOrEmpty(expanderContext.AreaName) ?
-                AreaViewLocationFormats :
-                ViewLocationFormats;
+                _options.AreaViewLocationFormats.AsEnumerable() :
+                _options.ViewLocationFormats.AsEnumerable();
```

Comment:
What are the `AsEnumerable()` calls for? If you want to ensure the variable has type `IEnumerable<string>`, declare it with that type -- one of the few cases where not-`var` makes sense.